### ISSUE TITLE
Little trouble with the bounds of maps

### ISSUE THREAD

**Issue by CyrilCarmien** [Leaflet/Leaflet#3875]
Hi guys! First thanks for this awesome plugin =)
I'm having a small issue with the bounds of the markers. I think that when they're widespread all over the world they might be a problem somehow. Here's what it looks like:

![leaflet map bounds world](https://cloud.githubusercontent.com/assets/11183925/10157023/b9ad1896-66c9-11e5-873c-3a5a62517897.png)

And that's what it should look like:

![leaflet map bounds should be](https://cloud.githubusercontent.com/assets/11183925/10157024/b9af0066-66c9-11e5-8853-8197f4d0aa11.png)

I'm pretty sure I'm using the bounds correctly as for another map where there is only results in Australia and Asia I have this:

![leaflet map bounds](https://cloud.githubusercontent.com/assets/11183925/10157022/b9ad28ea-66c9-11e5-9097-159847f1487e.png)

Here's bound part of my code: 

    var group = new L.featureGroup(bounds);
        markers.addLayers(bounds);
        map.addLayer(markers);

        map.fitBounds(group.getBounds().pad(0.5));
    </script>

Knowing that bounds is an array of L.marker

Cheers!

**Comment by mourner** [comment]
Can you set up a minimal playground-leaflet.rhcloud.com or JSFiddle test case reproducing the issue (with just Leaflet, no plugins)?

**Comment by CyrilCarmien** [comment]
Hi,
Sorry about that but when I was actually writing that minimal js without sensitive data I actually found out what the issue was, that .pad(0.5) was too big for the world map to fit. I've reduced it to 0.2 and now it's displaying correctly. What I think the issue was is that I have markers in North America and Australia which are basically opposite to one another, and the padding around these 2 was bigger than the world map so it displayed several maps instead of one?

Thanks anyways!

![leaflet map bounds](https://cloud.githubusercontent.com/assets/11183925/10180459/c09b486c-6750-11e5-9175-c83315797d66.png)
